server/src/database/gameDB.ts

```diff
@@ -71,15 +75,20 @@ function createGame (): BaseUnifiedGame {
     playerPlayerStatuses: {},
     playerNotes: {},
     deadVotes: {},
+    travelers: {},
   }
 }
 
 export function addPlayer (
   gameId: string,
   player: string,
+  traveler?: boolean,
 ): void {
   const game = retrieveGame(gameId)
   const gameInstance = game.readOnce()
+  const gameStarted = gameInProgress(gameInstance)
+  // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+  const travelling = traveler || gameStarted
```

Comment:
Extra 'L' typo

Reply:
fun fact, the wiki also has this typo several times